platform/build-faiss/build-faiss.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+git clone https://github.com/facebookresearch/faiss
+cd faiss
+
+# Configure paths and set environment variables
+export PATH=$PATH:$HOME/.local/bin:/usr/local/cuda/bin
+source /opt/intel/oneapi/setvars.sh
+
+#export CC=gcc-12
+#export CXX=g++-12
+# Configure using cmake
+
+LD_LIBRARY_PATH=/usr/local/lib MKLROOT=/opt/intel/oneapi/mkl/2023.1.0/ CXX=g++-11 cmake -B build \
```

Comment:
A reference to the cflags would help. To do that, I would insert ahead of line 14:

# https://github.com/facebookresearch/faiss/blob/main/INSTALL.md#step-1-invoking-cmake